src/nodes.cpp

```diff
@@ -65,31 +76,55 @@ Selection NodePopulation::matchAttributeValues(const std::string& name, const T
     }
 }
 
+template <typename T>
+Selection NodePopulation::matchAttributeValues(const std::string& name, const T value) const {
+    std::vector<T> values{value};
+    return matchAttributeValues<T>(name, values);
+}
+
 template <>
-Selection NodePopulation::matchAttributeValues<std::string>(const std::string& name,
-                                                            const std::string value) const {
-    // enum attribute, need to look up in the @library
+Selection NodePopulation::matchAttributeValues<std::string>(
+    const std::string& name, const std::vector<std::string>& values) const {
     if (enumerationNames().count(name) > 0) {
         const auto enum_values = enumerationValues(name);
-        const auto pos = std::find(enum_values.begin(), enum_values.end(), value);
-        if (pos == enum_values.end()) {
+        std::vector<size_t> wanted_enum_value;
+        wanted_enum_value.reserve(values.size());
+        for (const auto& v : values) {
+            const auto wanted_index = std::find(enum_values.cbegin(), enum_values.cend(), v);
```

Comment:
So worst case here is `n log(n)` string comparisons if we want to match close to all values… would a look-up table be warranted? That would give `2n` with a bunch of additional memory consumption.